Pcap++/CMakeLists.txt

```diff
@@ -2,8 +2,8 @@ add_library(
   Pcap++
   $<$<BOOL:${PCAPPP_USE_DPDK}>:src/DpdkDevice.cpp>
   $<$<BOOL:${PCAPPP_USE_DPDK}>:src/DpdkDeviceList.cpp>
-  $<$<BOOL:${PCAPPP_USE_DPDK}>:src/KniDevice.cpp>
-  $<$<BOOL:${PCAPPP_USE_DPDK}>:src/KniDeviceList.cpp>
+#  $<$<BOOL:${PCAPPP_USE_DPDK}>:src/KniDevice.cpp>
```

Comment:
why dropping these files ? at least it should be compatible with 22.11

Reply:
@SDMProjects yes, I agree with @clementperon - why do we need to remove KNI? 

According to this: https://doc.dpdk.org/guides/rel_notes/deprecation.html KNI will only be removed in DPDK 23.11 🤔